packages/protocol/test/governance/vesting.ts

```diff
@@ -0,0 +1,1734 @@
+    await vestingFactoryInstance.initialize(registry.address)
+    await accountsInstance.initialize(registry.address)
+    await accountsInstance.createAccount({ from: beneficiary })
+
+    // prefund the vesting factory instance with 2 gold tokens to simulate a well-funded factory contract in the genesis block
+    await goldTokenInstance.transfer(vestingFactoryInstance.address, ONE_GOLDTOKEN.times(2), {
+      from: owner,
+    })
+  })
+
+  describe('#initialize()', () => {
+    it('should set the owner', async () => {
+      const vestingFactoryOwner: string = await vestingFactoryInstance.owner()
+      assert.equal(vestingFactoryOwner, owner)
+    })
+
+    it('should set the registry address', async () => {
+      const registryAddress: string = await vestingFactoryInstance.registry()
+      assert.equal(registryAddress, registry.address)
+    })
+
+    it('should revert if already initialized', async () => {
+      await assertRevert(vestingFactoryInstance.initialize(registry.address))
+    })
+  })
+
+  describe('#setRegistry()', () => {
+    const anAddress: string = accounts[2]
+
+    it('should set the registry when called by the owner', async () => {
+      await vestingFactoryInstance.setRegistry(anAddress)
+      assert.equal(await vestingFactoryInstance.registry(), anAddress)
+    })
+
+    it('should revert when not called by the owner', async () => {
+      await assertRevert(vestingFactoryInstance.setRegistry(anAddress, { from: beneficiary }))
+    })
+  })
+
+  describe('#payable()', () => {
+    it('should accept gold transfer by default from anyone', async () => {
+      await createNewVestingInstanceTx(vestingDefaultSchedule, web3)
+      const vestingInstanceRegistryAddress = await vestingFactoryInstance.vestings(beneficiary)
+      await goldTokenInstance.transfer(vestingInstanceRegistryAddress, ONE_GOLDTOKEN.times(2), {
+        from: accounts[8],
+      })
+    })
+  })
+
+  describe('#vesting - creation()', () => {
+    it('should fail if a non-owner attempts to create a vesting instance', async () => {
+      const vestingSchedule = _.clone(vestingDefaultSchedule)
+      vestingSchedule.vestingStartTime = (await getCurrentBlockchainTimestamp(web3)) + 5 * MINUTE
+      await assertRevert(
+        vestingFactoryInstance.createVestingInstance(
+          vestingSchedule.vestingBeneficiary,
+          vestingSchedule.vestingNumPeriods,
+          vestingSchedule.vestingCliff,
+          vestingSchedule.vestingStartTime,
+          vestingSchedule.vestingPeriodSec,
+          vestingSchedule.vestAmountPerPeriod,
+          vestingSchedule.vestingRevocable,
+          vestingSchedule.vestingRevoker,
+          vestingSchedule.vestingMaxPausePeriod,
+          { from: accounts[5] }
+        )
+      )
+    })
+
+    it('should revert if a vesting instance to the beneficiary already exists', async () => {
+      await assertRevert(createNewVestingInstanceTx(vestingDefaultSchedule, web3))
```

Comment:
I have reworked this and actually used a "beforeEach" a bit higher, making sure that each it will have a freshly new created instance. In that particular test a vesting instance is already there and we try to create a new one to that beneficiary, hence it fails.